### DISCUSSION TITLE
error Error occur while filtering : Attempting to open an ingest buffer caused by: could not open ingest caused by: ingest buffer does not exist:

### DISCUSSION THREAD

**Discussion by geraldkenny** [datalust/seq-tickets#1779] [Help]
Hello, please I am currently encountering this issue while trying to filter my logs on seq dashboard. Seq version 2023. 1.8991 running on azure web app service with azure file storage. Any help would be appreciated.

**Comment by nblumhardt** [comment]
Hi Gerald, thanks for reaching out.

Unfortunately, Seq [doesn't support Azure Web Apps](https://docs.datalust.co/docs/using-azure-websites-or-app-service) or non-Premium Azure Files Storage.

Regarding web apps, this is because Azure doesn't guarantee that a single instance of the container will run, and because there is no reliable file locking mechanism available, the multiple Seq containers will corrupt each other's storage.

In the case of Azure Files storage, the underlying technology is not compatible with Seq's I/O requirements (as far as we understand it, this problem is shared with other kinds of database technologies like PostgreSQL and SQLite, which also don't run reliably on Azure Files).

You can repair the error in the title by creating an [init script](https://docs.datalust.co/docs/init-scripts) with:

```
flaretl repair --metadata /data/Stream
```

Once your stream is recovered, however, you'll need to move it to a supported tech stack in order to prevent more (and potentially unrecoverable) corruptions.

For serverless hosting on Azure, we've had good success with [Azure Container Instances + Azure Premium Files](https://docs.datalust.co/docs/using-azure-container-instances-aci) storage, or if you have access to it, [Azure Kubernetes Services + Azure Disks](https://docs.datalust.co/docs/using-aks).

Hope this helps,
Nick
